packages/migrator/src/utils/kvdb/addAppPrefix.spec.ts

```diff
@@ -0,0 +1,195 @@
+import { faker } from '@faker-js/faker'
+import IORedis from 'ioredis'
+import { GenericContainer } from 'testcontainers'
+
+import { addAppPrefix } from './addAppPrefix'
+
+import type { Redis } from 'ioredis'
+import type { StartedTestContainer } from 'testcontainers'
+
+import { getAppPrefixedKey } from '@/utils/kvdb/keyPrefix'
+import { generateAppKeys, fillKVDB, getNonMatchingCount, getAllKeys } from '@/utils/tests/kvdb'
+
+
+const STARTUP_TIMEOUT_IN_MS = 5 * 60 * 1000 // 5 min
+const CLEANUP_TIMEOUT_IN_MS = 30 * 1000 // 30 sec
+// NOTE: It's expensive to keep all keys in memory, so first const is used for tests with equality, second - for scans
+const SAFE_KEYS_SIZE = 100_000
+const SAFE_KEYS_TIMEOUT_IN_MS = 30_000 // 30 sec is enough to fill db / run test / scan db on 100k keys scale
+const BIG_KEYS_SIZE = 3_000_000
+const BIG_KEYS_TIMEOUT_IN_MS = 60 * 1000 // 1 min
+
+let currentDB = -1
+
+function getRedisVirtualDBConnection (host: string, port: number): string {
+    const dbIndex = ++currentDB
+    return `redis://${host}:${port}/${dbIndex}`
+}
+
+const TASKS_QUEUES = ['tasks', 'low', 'high']
+
+function randInt (min: number, max: number): number {
+    return Math.floor(Math.random() * (max - min)) + min
+}
```

Comment:
Why not use a ready-made one if it is already installed?

```faker.datatype.number({min: 10, max: 50})```

Reply:
Had no idea this existed :)

So decided to write my own one-liner